docs/management/snapshot-repositories/index.asciidoc

```diff
@@ -0,0 +1,135 @@
+begin with an overview of your repositories.  You can then use the UI to 
+drill down into your repositories and snapshots.
+
+[role="screenshot"]
+image:management/snapshot-repositories/images/repository_list.png["Repository list"]
+
+[float]
+=== Registering a repository
+
+You must have a registry before you can save your snapshots. If you don’t see a 
+repository in the list, navigate to *Register repository* to create one. 
+
+[role="screenshot"]
+image:management/snapshot-repositories/images/register_repo.png["Register repository"]
+
+{kib} supports three types of repositories on startup:
+
+* *Shared file system.*  Uses a shared file system to store the snapshots.
+* *Read-only url.*  Provides a read-only alternative to access snapshot data 
+in another repository. 
+* *Source-only.*  Enables storage of minimal, source-only snapshots.  
+A source-only repository can take up to 50% less disk space.
+
+For more information on these repositories and the settings that you can 
+configure, see {ref}/modules-snapshots.html#_repositories[Repositories]. 
+To add support for additional types, see 
+{ref}/modules-snapshots.html#_repository_plugins[Repository plugins].
+
+A best practice is to register a repository for each major version of 
+{es}. If you register the same snapshot repository with multiple clusters, 
+give only one cluster write access to the repository. All other clusters 
+connected to that repository should have read-only access.
+
+[float]
+=== Creating a snapshot
+
+Use the Kibana <<console-kibana, Console>> to create your snapshots. The 
+{ref}//modules-snapshots.html#_snapshot[snapshot API]
+takes the current state and data in your index or cluster and saves it to a 
+shared repository. 
+
+The snapshot process is "smart." Your first snapshot is a complete copy of data.
+All subsequent snapshots save the changes between the existing snapshots and 
+the new data.
+
+For an overview of the snapshots in your cluster, go to the *Snapshots* tab.
+You can then drill down into the details for each snapshot.
+
+[role="screenshot"]
+image:management/snapshot-repositories/images/snapshot_details.png["Snapshot details"]
+
+[float]
+=== Example: Register a shared file system repository
+
+This example shows how to register a shared file system repository 
+and store snapshots.
+
+[float]
+==== Register the repository location
+
+You must register the location of the repository in the `path.repo` setting on 
+your master and data nodes.  You can do this in one of two ways:
+
+* Edit  your `elasticsearch.yml` to include the `path.repo` setting.
+
+* Pass the `path.repo` setting when you start Elasticsearch.
++
+`bin/elasticsearch -E path.repo=/tmp/es-backups`
+
+[float]
+==== Register the repository
+
+Use *Snapshot Repositories* to register a repository. 
+
+
+. Go to *Management > Elasticsearch > Snapshot Repositories*.
+. Click *Register a repository*.
+. Enter a name for your repository. For example, `my_backup.`
```

Comment:
Super knit: Should the period go after `? So it would look like:

For example, `my_backup`.